Run Godot 4.4 Upgrade uids...

jamie-pate opened the pull request:
Godot 4.4. uids are viral and must also be in addons

## What kind of change does this PR introduce?

Upgrade compatibility to Godot 4.4

## What is the current behavior?

Godot 4.4 projects will constantly update the addon with new uids if they are not present

## What is the new behavior?

UIDs are committed as is required by Godot 4.4

## Additional context

[Add any other context or screenshots.](https://godotengine.org/article/uid-changes-coming-to-godot-4-4/)

fenix-hub commented:
Nice! Very thankful to everybody for this. Honestly I was a little bit afraid of this update since I'm not opening godot since 4.0 and felt a little bit overwhelmed. I'll make sure to educate myself on the new update and integrate your changes. thanks @jamie-pate and @namelessvoid .

jamie-pate commented:
weird, idk how those changes happened, my git history looked like I took 464e1f4532ffcb0f3796321e72c12d4d60b267be and just added the uids on top :shrug: 

anyways, i've updated it starting from 464e1f4532ffcb0f3796321e72c12d4d60b267be again.. project.godot probably should actually be in the git repo though.. godot addons be weird.

namelessvoid reviewed (approved):
Thanks for fixing the PR, and no worries, Git fools us all from time to time :grin:
